.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,113 @@
+name: Testsuite
+
+on: [push]
+
+jobs:
+  flake8:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version:  [3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Get pip cache dir
+      id: pip-cache
+      run: |
+        echo "::set-output name=dir::$(pip cache dir)"
+    - name: pip cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.pip-cache.outputs.dir }}
+        key: py${{ matrix.python-version }}-${{ matrix.os }}-pip
+
+    - name: Syntax errors check
+      run: |
+        python -m pip install flake8
+        flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
+    - name: Style warnings
+      run: |    
+        # exit-zero treats all errors as warnings.
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=79 --statistics        
+
+  build:
+    needs: flake8
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 4
```

Comment:
I think the default would be better (e.g. larger). We could easily have all (currently 6) build jobs build in parallel. Maybe just put 8 here...